### ISSUE TITLE
Top-right “create” button on the roomlist view is unlabelled for voiceover

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-x-ios#3701]
### Steps to reproduce

1. use voiceover
2. apparently the “create” button bas no label

See https://dragonscave.space/@gregarious/113880412719300513

#### What did you expect?

We should have good screenreader support via VoiceOver

#### What happened instead?

Unlabelled button

### Will you send logs?

No

**Comment by ara4n** [comment]
More context:

> I saw a mention of this on mastodon, and also included a screen recording, indicating the exact item I was referring to which even included speech from voiceover. And turns out the item I was referring to maybe possibly associated with the search box that’s in between the settings and start chat buttons. The only way to actually access the unlabeled button is flicking left to right using voiceover, as it’s not accessible when using the direct touch method meaning putting your finger on a specific location on the screen. That’s why I included the screen recording so I could make that particular point clear.

https://dragonscave.space/@gregarious/113880674251332196

https://github.com/user-attachments/assets/0c63ed97-cf29-49cf-943a-9df5e59483e9
